cypress/integration/ennexos.sunnyportal.spec.ts

```diff
@@ -1,41 +1,47 @@
 /// <reference types="cypress" />
 
 import { SMA_USERNAME, SMA_PASSWORD, SMA_PLANT_ID } from '../../credentials';
+import { DashboardPage } from '../page-models/dashboard.page';
+import { NavigationModel } from '../page-models/navigation.model';
+import { EnergyBalancePage } from '../page-models/energy-balance.page';
+import { LoginPage } from '../page-models/login.page';
 
 describe('Visit ennexos.sunnyportal.com', () => {
   const username = SMA_USERNAME;
   const password = SMA_PASSWORD;
-  const plantId = SMA_PLANT_ID
+  const plantIds = SMA_PLANT_ID;
 
   beforeEach(() => {
-    cy.visit('https://ennexos.sunnyportal.com');
+    cy.visit(Cypress.env('BASE_URL'));
+
+    cy.get('#onetrust-accept-btn-handler').then((btn) => btn.trigger('click'));
   });
 
   it('redirects to /dashboard on success', () => {
-
     // Login
     cy.url().should('include', '/login');
-    cy.get('input[name=username]').type(username);
-    cy.get('input[name=password]').type(password);
-    cy.get('form').submit();
 
-    // Got to Dashboard
-    cy.url().should('include', `${plantId}/dashboard`);
-    cy.get('h1').should('contain', 'Dashboard');
+    new LoginPage().login({username, password});
 
-    // Click on Monitoring
-    cy.get('#ennexos-element-monitoring').click();
-    cy.get('a[href*="monitoring/view-energy-balance"]').click()
+    cy.url().should('include', '/dashboard');
 
-    // Select "Details" and click on "Download" button
-    cy.get('span.ennexos-accordion-text').contains('Details').click();
-    cy.get('button.btn.ennexos-button').click();
+    const isMultiPlant = plantIds.includes(',');
```

Comment:
plantIds is a number

Reply:
see https://github.com/thorsten/sunnyportal-csv-downloader/blob/main/credentials.original.ts#L8